.maestro/testsV2/subFolder/links/DeepLinksGeneric.yml

```
@@ -0,0 +1,21 @@
+appId: app.passculture.webapp
+---
+- runFlow:
+    when:
+      true: ${MAESTRO_APP_INSTALLED == 'true'}
+    commands:
+      - stopApp
+      - openLink:
+          link: ${URL}
+          autoVerify: true
+      - runFlow: ../../subFolder/cookies/AcceptCookiesOptional.yml
+
+- runFlow:
+    when:
+      true: ${MAESTRO_APP_INSTALLED == 'false'}
+    commands:
+      - stopApp: com.android.chrome
+      - openLink:
+          link: ${URL}
+          autoVerify: true
+      - runFlow: ../../subFolder/cookies/AcceptCookiesOptional.yml
```

il me semble qu'on peut mutualiser l'ouverture de lien et l'acceptation des cookies

```suggestion

- runFlow:
    when:
      true: ${MAESTRO_APP_INSTALLED == 'false'}
    commands:
      - stopApp: com.android.chrome

- openLink:
    link: ${URL}
    autoVerify: true
- runFlow: ../../subFolder/cookies/AcceptCookiesOptional.yml
```

---

Donc mettre l'ouverture des liens + l'acceptation des cookies dans un seul flow ?